packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -9,8 +9,8 @@ import {
 	WorkflowEdge,
 	WorkflowNode,
 	WorkflowPortStatus,
-	WorkflowStatus
-} from '@/types/workflow';
+	OperatorStatusType
```

Comment:
I thought `Type` would be a good idea because of this (in this case the StatusEnum is OperatorStatusType):
![image](https://github.com/DARPA-ASKEM/terarium/assets/28237258/cc950bbc-90b9-4c71-9f1f-4badc95754ba)

Reply:
I'm so sorry, that note document is just a scratch pad for arguing about a solution, not a thoughtful coding plan.

What you screenshot should be more like for now:
```json
"status" : {
    "type": OperatorStatus,
    "message": string,
    "metadata": any,
}
```